### PR TITLE
fix/i18n-de

### PR DESCRIPTION
**Pull request by DJCrashdummy**
fixed, polished and updated the german translation.

### CODE REVIEW

File: frontend/locale/de.json [hunk 1]
```diff
@@ -17,65 +17,65 @@
...
   "nav.login.title": "Login",
-  "nav.accountmenu.title": "Mein Account",
+  "nav.accountmenu.title": "Mein Benutzerkonto",
   "nav.accountmenu.logout": "Ausloggen",
-  "nav.accountmenu.admin": "Mein Account",
+  "nav.accountmenu.admin": "Mein Benutzerkonto",
   "nav.brand": "Open Eats",
-  "nav.news": "News",
-  "nav.recipes": "Rezepte",
+  "nav.news": "Neues",
```
Comment (DJCrashdummy) [inline frontend/locale/de.json]: well i'm not sure if this is the correct translation for "nav.news"...
- now it is like "is there something new" (= "Neues")
- or should it be like "news in the newspaper" (= "Nachrichten")?

File: frontend/locale/de.json [hunk 2]
```diff
@@ -17,65 +17,65 @@
...
+  "recipe.import.error_message_2": "Diese Webseite wird nicht unterstützt.",
+  "recipe.import.error_message_3": "Bitte eine zulässige URL eingeben.",
+  "recipe.import.hide_sites": "unterstützte Seiten verbergen",
+  "recipe.import.show_sites": "unterstützte Seiten anzeigen",
   "recipe.create.name_label": "Rezepttitel",
   "recipe.create.name_placeholder": "Rezept",
   "recipe.create.course_label": "Gang",
   "recipe.create.cuisine_label": "Küche",
-  "recipe.create.tags_label": "Tags",
+  "recipe.create.tags_label": "Schlagworte",
```
Comment (DJCrashdummy) [inline frontend/locale/de.json:78]: i translated "recipe.create.tags_label" to match "searchbar.placeholder"...
although "tag" is also used as anglicism in the german language, some tech-unsavvy german native speakers have no idea what "tags" are.

File: frontend/locale/de.json (hunk 1 above)
Comment (RyanNoelk) [reply frontend/locale/de.json]: It's ambiguous. It depends on how the user whats to use it. I personally would take it to me as   "news in the newspaper". But each person might have a different interpretation.

**Comment by RyanNoelk** [comment]
Translations LGTM, Looks like the translation of OpenEats is causing some conflicts. Once that is updated, I'll merge this in.

File: frontend/locale/de.json (hunk 1 above)
Comment (DJCrashdummy) [reply frontend/locale/de.json]: i changed it back to "News", as it is also used as anglicism... and so it still has both meanings.

**Comment by DJCrashdummy** [comment]
sorry, i'm pretty sure it was my fault because i forked both of my branches from your "feature/i18-update" first and then made my changes.
...i'm still not an expert in git and github! :-/

**Review by RyanNoelk — APPROVED** [review]
LGTM

**Pull request merged**